package.json

```diff
@@ -4,11 +4,29 @@
   "description": "Brave Core is a set of changes, APIs, and scripts used for customizing Chromium to make Brave.",
   "main": "index.js",
   "scripts": {
-    "sync": "npm --prefix ../../ run sync --",
-    "build": "npm --prefix ../../ run build --",
-    "start": "npm --prefix ../../ run start --",
-    "test": "npm --prefix ../../ run test --",
-    "lint": "tslint --project tsconfig-lint.json \"components/**/*.{ts,tsx}\"",
+    "audit_deps": "node ./build/commands/scripts/audit.js",
+    "cibuild": "node ./build/commands/scripts/commands.js cibuild",
+    "init": "npm --prefix ../../ run init --",
+    "delete_string_translations": "node ./build/commands/scripts/commands.js delete_string_translations",
+    "create_dist": "node ./build/commands/scripts/commands.js create_dist",
+    "sync": "node ./build/commands/scripts/sync.js",
+    "gclient": "node ./build/commands/scripts/gclient.js",
+    "build": "node ./build/commands/scripts/commands.js build",
+    "gn_check": "node ./build/commands/scripts/commands.js gn_check",
+    "versions": "node ./build/commands/scripts/commands.js versions",
+    "upload": "node ./build/commands/scripts/commands.js upload",
+    "update_patches": "node ./build/commands/scripts/commands.js update_patches",
+    "apply_patches": "node ./build/commands/scripts/commands.js apply_patches",
+    "start": "node ./build/commands/scripts/commands.js start",
+    "network-audit": "node ./build/commands/scripts/commands.js start --enable_brave_update --network_log --user_data_dir_name=brave-network-test --disable-doh",
+    "push_l10n": "node ./build/commands/scripts/commands.js push_l10n",
+    "pull_l10n": "node ./build/commands/scripts/commands.js pull_l10n",
+    "chromium_rebase_l10n": "node ./build/commands/scripts/commands.js chromium_rebase_l10n",
+    "lint": "node ./build/commands/scripts/commands.js lint",
+    "test": "node ./build/commands/scripts/commands.js test",
+    "test:scripts": "jest build/commands/lib build/commands/scripts",
+    "test-security": "npm run audit_deps && node ./build/commands/scripts/commands.js start --enable_brave_update --network_log --user_data_dir_name=brave-network-test",
+    "tslint": "tslint --project tsconfig-lint.json \"components/**/*.{ts,tsx}\"",
```

Comment:
Looks like this change by necessity allows C++ lints (`npm run lint`) and typescript lints (`npm run tslint`) to be run independently from the same directory, which is awesome.

It does seem a little weird to me to have a specifically-named command `tslint` and generically-named command `lint` with similar behavior for different languages; I'd recommend something like the following:
- `npm run tslint` as specified above
- `npm run cpplint` to take the current behavior of `npm run lint`
- Optionally, also `npm run lint` as a convenient way to run all of the above together

This scheme would be pretty natural to extend to other languages too, e.g. for our Rust FFIs.

 I'd also like to see these aliased the same way in `brave-browser` (currently `tslint` is missing from https://github.com/brave/brave-browser/pull/10885).

Reply:
it's just for backwards compat to get this through CI and actually I want to get rid of all the aliases commands in brave-browser which are also there for backwards compat. This is just the first step in moving these files